### PR TITLE
SC-5265 - lernstore-arrow-back-window

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
changed reuirements, right now it should be full size window with arrow to close it
please check also client PR https://github.com/schul-cloud/schulcloud-client/pull/1988

## Links to Tickets or other pull requests

- https://github.com/schul-cloud/schulcloud-client/pull/1988
- https://ticketsystem.schul-cloud.org/browse/SC-5265

## Screenshots of UI changes
image

### CODE REVIEW

File: src/pages/content/index.vue [hunk 1]
```diff
@@ -101,6 +110,9 @@ export default {
 			}
 			return query;
 		},
+		isInline() {
+			return window.location.search.includes("inline=1");
```
Comment (BlackGhostHD) [inline src/pages/content/index.vue:114]: use $route object instead of window;  this.$route.query.inline

File: src/pages/content/index.vue [hunk 2]
```diff
@@ -63,6 +70,8 @@ import infiniteScrolling from "@mixins/infiniteScrolling";
 import BaseGrid from "@components/base/BaseGrid";
 import ContentEduSharingFooter from "@components/molecules/ContentEduSharingFooter";
 
+import Theme from "@theme/config";
```
Comment (BlackGhostHD) [inline src/pages/content/index.vue:73]: You do not need to do that anymore, it is already imported into the global.js. You can directly use $theme to get for example: the short_name

Comment (BlackGhostHD) [inline src/pages/content/index.vue:73]: You do not need to do that anymore, it is already imported into the global.js. You can directly use $theme to get for example: the short_name

File: src/pages/content/index.vue [hunk 3]
```diff
@@ -169,11 +181,23 @@ export default {
...
 	head() {
-		return {
-			title: "LernStore",
-		};
+		if (this.isInline) {
+			return {
+				title:
+					this.$t("pages.content.page.window.title") +
+					Theme.name +
+					this.$t("pages.content.page.window.title_2"),
```
Comment (BlackGhostHD) [inline src/pages/content/index.vue:194]: https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting

File: src/pages/content/index.vue (hunk 1 above)
Comment (BlackGhostHD) [inline src/pages/content/index.vue:114]: use $route object instead of window;  this.$route.query.inline

File: src/pages/content/index.vue [hunk 4]
```diff
@@ -1,5 +1,12 @@
 <template>
 	<section>
+		<base-icon
```
Comment (BlackGhostHD) [inline src/pages/content/index.vue:3]: I think, but I'm not sure, for the sake of Accessibility a button would be better here

File: src/pages/content/index.vue [hunk 5]
```diff
@@ -169,11 +181,23 @@ export default {
...
 		},
+		goBack() {
+			window.close();
+		},
 	},
 	head() {
-		return {
-			title: "LernStore",
-		};
+		if (this.isInline) {
```
Comment (BlackGhostHD) [inline src/pages/content/index.vue:189]: You could use a ternary operator here

File: src/pages/content/index.vue (hunk 1 above)
Comment (ABialoskorski) [reply src/pages/content/index.vue:114]: done ;)

File: src/pages/content/index.vue (hunk 2 above)
Comment (ABialoskorski) [reply src/pages/content/index.vue:73]: done ;)

File: src/pages/content/index.vue (hunk 3 above)
Comment (ABialoskorski) [reply src/pages/content/index.vue:194]: nice ;)

File: src/pages/content/index.vue (hunk 5 above)
Comment (ABialoskorski) [reply src/pages/content/index.vue:189]: done ;)

File: src/pages/content/index.vue (hunk 4 above)
Comment (ABialoskorski) [reply src/pages/content/index.vue:3]: added ;)

File: src/pages/content/index.vue [hunk 6]
```diff
@@ -1,5 +1,8 @@
 <template>
 	<section>
+		<base-button v-if="isInline" design="none" type="button" @click="goBack">
+			<base-icon class="arrow__back" source="material" icon="arrow_back" />
+		</base-button>
```
Comment (BlackGhostHD) [inline src/pages/content/index.vue:5]:
```suggestion
		<base-button
			v-if="isInline"
			design="text icon"
			type="button"
			class="arrow__back"
			@click="goBack"
		>
			<base-icon source="material" icon="arrow_back" />
		</base-button>
```
design="none" will not have any effects, e.g: thier will be not cursor: pointer

Comment (ABialoskorski) [reply src/pages/content/index.vue:5]: done ;)

**Comment by ABialoskorski** [comment]
> Unfortunately doesn't work with multiple screens.

fixed on client PR

**Pull request merged**